src/reducers/player.js

```diff
@@ -1,19 +1,44 @@
 import {ActionTypes} from '../constants';
-const {INITIAL_STATE} = ActionTypes;
+const {INITIAL_STATE, TILE_SELECTED} = ActionTypes;
 
 export default function player(state = {}, action) {
   if (action.turn !== state.id && action.type !== INITIAL_STATE) {
     return state;
   }
 
   switch (action.type) {
+    case TILE_SELECTED:
+      if (state.selected == '') {
```

Comment:
generally we use `===` in javascript because `==` doesn't check the type of the values being compared